apps/nova/src/lib/ratings.ts

```diff
@@ -2,11 +2,13 @@ import { IRating } from "@litespace/types";
 import { TutorRatingCardGroupProps } from "@litespace/luna/TutorFeedback";
 
 export const organizeRatings = (
-  ratings: IRating.FindTutorRatingsApiResponse["list"],
+  ratings: IRating.FindTutorRatingsApiResponse["list"] | undefined,
   currentUserId: number | undefined
 ) => {
-  let currentUserRating: IRating.RateeRatings | undefined;
-  const ratingsWithFeedback: IRating.RateeRatings[] = [];
+  if (!ratings) return [];
+
+  let currentUserRating: IRating.RateeRating | undefined;
+  const ratingsWithFeedback: IRating.RateeRating[] = [];
   const ratingsWithoutFeedback: Omit<TutorRatingCardGroupProps, "tutorName">[] =
```

Comment:
Let's not use the props here and use `IRating.RateeRating` for consistency and then you can map it in the component.